### PR TITLE
Bug/moz 317 community links

### PR DESCRIPTION
**Pull request by kycapstick**
Updated Community Links in both profile and group to handle missing "http"

### CODE REVIEW

File: lib/utils.php [hunk 1]
```diff
@@ -250,4 +250,12 @@ function mozilla_redirect_admin() {
     }
 }
 
+function mozilla_verify_url($url) {
```
Comment (dnaluz) [inline lib/utils.php]: We need to also handle https

**Comment by dnaluz** [comment]
@kycapstick I have added to the conversation here https://github.com/mozilla/community-portal/issues/236.  Let's see what the response is if they want to move away from allowing users to put in just their user names or not, we could still do some sort of validation if we check the string they are giving us, you can check if the TLD and the SLD are present for the specific site in question and if not we assume it's just the username.  Let's see what the response is from Mozilla before moving forward.

**Comment by dnaluz** [comment]
@kycapstick what happened to this branch?  Looks like this branch was deleted before it was merged in?  Did we fix these issues?

**Comment by kycapstick** [comment]
Sorry @dnaluz, realized I had done some work on the image stuff here, and following our discussions on this, wanted to move the fix to a clean branch so I could only move over what was needed rather than try and pick through. Will be submitting a fresh PR with those changes.

**Pull request closed without merging**